modules/ROOT/pages/syntax/parameters.adoc

```diff
@@ -15,14 +15,15 @@ Additionally, parameters make caching of execution plans much easier for Cypher,
 
 Parameters can be used for:
 
-* literals and expressions
-* node and relationship ids
+* Literals and expressions.
+* Node and relationship ids.
+* Node labels and relationship types, when referenced dynamically (for more information, see xref:clauses/match.adoc#dynamic-match[`MATCH` using dynamic node labels and relationship types]). label:new[Introduced in 5.26]
```

Comment:
Do you want to mention they can be used for properties when dynamically referenced too? https://neo4j.com/docs/cypher-manual/current/clauses/where/#filter-on-dynamic-property